src/Makefile.legacy

```diff
@@ -1759,10 +1757,6 @@ $(SUBDIRS):
 ../run/unique.com: john.com
 	copy john.com ..\run\unique.com
 
-# This must not be specified with path because "make clean" would delete it.
-john.local.conf:
-	[ -f ../run/john.local.conf ] || touch ../run/john.local.conf
-
```

Comment:
I just compared 2 different clones. after I used the first one for an autoconf build and `make -s distclean` and the second one for a legacy build and `make -s -f Makefile.legacy clean`.

The other difference between those 2 clones is an empty `run/kernels` directory in the autoconf clone. 
Apparently, this directory is created even for CPU-only builds.
(May be we should always remove run/kernels on `make distclean`, but don't create it for CPU builds.)